src/content/guides/code-splitting.md

```diff
@@ -197,6 +197,7 @@ __webpack.config.js__
       filename: '[name].bundle.js',
 +     chunkFilename: '[name].bundle.js',
       path: path.resolve(__dirname, 'dist')
+      publicPath: 'dist/'
```

Comment:
Does it work removing `/`, I think it is unnecessary.

Reply:
No, in my example it has to be ``dist/`` for it to work.

Reply:
Win 10. Sorry for the late reply.